pkg/build/runner.go

```diff
@@ -67,73 +67,76 @@ func (r *BuildRunner) Stop() {
 }
 
 func (r *BuildRunner) Run() {
-	pendingState := BuildStatePending
-	builds, err := r.buildStore.List(&BuildFilter{State: &pendingState})
+	builds, err := r.buildStore.List(nil)
```

Comment:
I suggest we filter by pending and success state in this case (either we make the `State` filter an array or you fetch them separately).
Getting all builds might be expensive as the builds table grows